### DISCUSSION TITLE
Should composite script print all commands or just an  empty list?

### DISCUSSION THREAD

**Discussion by monchin** [pdm-project/pdm#3318] [Ideas]
As in 2.21.0, I found that defaultly the console would output "Running <task {my task}> [{args}]" when running pdm script. However, if the task is a composite task, the args list would be an empty list. Should the args list output all the commands? If so, should all commands be in one line or each command one line?

**Comment by frostming** [comment]
No, the output was a regression and will be reverted: #3313
